app/src/utils/dataFormatting.ts

```diff
@@ -0,0 +1,9 @@
+// Modified from https://stackoverflow.com/a/8358141
+export function formatPhone(phone: string) {
```

Comment:
Could be just me but in terms of readability a doc string might be super nice to explain the format / phone number type formatPhone expects to receive and return.

also absolute nit (🙏 ) but stripNonNumbers gives me cognitive dissonance b/c the name is more like the output not the function doing it 😬 maybe could just put // strip non numbers in a code comment then name the value like "parsedNumber" something, but I could just be being annoying!